packages/itwinui-css/src/bridge.scss

```diff
@@ -311,26 +430,50 @@
 
     // StrataKit tab has no background or border
     border: none;
+    background-color: transparent;
```

Comment:
The `background-color` was intentionally removed in #2541.

Reply:
Seems like this should be fixed outside of theme bridge. We can remove the background color for default Tabs.